cg/constants/housekeeper_tags.py

```diff
@@ -113,6 +113,18 @@ class BalsamicProtectedTags:
     ]
 
 
+class TaxprofilerProtectedTags(StrEnum):
+    """Taxprofiler workflow protected tags"""
+
+    KRAKEN2: str = "kraken2"
+    METAGENOMICS_REPORT: str = "metagenomics-report"
+    COMBINED_REPORT: str = "combined-report"
+    KRONA: str = "krona"
+    VISUALIZATION: str = "visualization"
+    MULTIQC_HTML: str = "multiqc-html"
+    MULTIQC_JSON: str = "multiqc-json"
```

Comment:
this class should be defined as `TaxProfilerAnalysisTags`

Reply:
But I'm also wondering if we can start with Taxprofiler to use the `storage` tag for long-term storage. It will help to maintain these tags and avoid the `legacy` tags that we have, for example, for mip-dna.

Similar approach but for storage that we've discussed here with @fevac: https://github.com/Clinical-Genomics/cg/issues/2848.

Reply:
Updated the PR.